boot-mongodb-elasticsearch/src/main/java/com/example/mongoes/web/controller/SearchController.java

```diff
@@ -25,24 +26,25 @@
 
 @RestController
 @Timed
-public class SearchController {
+@Validated
+class SearchController {
```

Comment:
@coderabbitai, can you please help me with adding validation to the requests and test validation using junits